docker-compose.yaml

```diff
@@ -95,6 +95,7 @@ services:
       - DEBUG_SKIP_API_KEY_CHECK=True
       - DEBUG_USE_SEED_DATA=True
       - MAX_WORKERS=1
+      - DEBUG_SKIP_EMBEDDING_COMPUTATION=True
```

Comment:
could you also add this to the ansible playbook?

Reply:
When deployed via ansible the fetching of the embeddings should _not_ be skipped?

Reply:
When should be skipped? Only in local-run.sh?

Reply:
That's why I asked again... probably also won't be necessary for Frontend development..